### PR TITLE
[13.0][ADD]product_category_tax

### PR DESCRIPTION
**Pull request by AaronHForgeFlow**
Allows to define taxes in the product category.

@ForgeFlow

PS: Do not merge yet, pending to solve this: https://github.com/OCA/account-financial-tools/pull/1091#issuecomment-733087543

### CODE REVIEW

**Comment by AaronHForgeFlow** [comment]
ping @LoisRForgeFlow I think this is ready :)

**Review by HviorForgeFlow — APPROVED** [review]

**Review by HaraldPanten — APPROVED** [review]
Functional review 👍

**Comment by LoisRForgeFlow** [comment]
Just a warning for merge masters, I know this is approved by 2 people but, please, hold on this until I am able to validate it, I want to check if the requirement from our customer is fully covered.

Thanks!

File: product_category_tax/models/product_category.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        string="Vendor Taxes",
+        help="Default taxes used when buying the product.",
+        domain=[("type_tax_use", "=", "purchase")],
+        default=lambda self: self.env.company.account_purchase_tax_id,
+    )
+    product_template_ids = fields.One2many(
+        "product.template", "categ_id", string="Products Templates"
+    )
+
+    def update_product_taxes(self):
```
Comment (LoisRForgeFlow) [inline product_category_tax/models/product_category.py:34]: this can be called on write, doesn't it? requiring a click is not very user friendly.

**Comment by LoisRForgeFlow** [comment]
Hi, this is good to merge now.

thanks!

File: product_category_tax/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Copyright 2020 ForgeFlow S.L. (https://www.forgeflow.com)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
```
Comment (rousseldenis) [inline product_category_tax/__manifest__.py:2]: @AaronHForgeFlow @JordiBForgeFlow I will migrate this. Is the license AGPL or LGPL as they are mixed through the module ?

Comment (AaronHForgeFlow) [reply product_category_tax/__manifest__.py:2]: I think AGPL is fine, as long as no other modules are built on top of this one.

Thank you.

**Pull request merged**